### PR TITLE
Fix incorrectly overridden project groups by `JvmTestSuite`.

### PR DESCRIPTION
**Pull request by ikhoon**
Motivation:

`group` was set in JvmTestSuite's configuration to change the test task's group, but `Project.group` was changed unintentionally. https://github.com/line/armeria/blob/e15c7b7809db6d77c6798c237707e942fceb51bb/core/build.gradle#L178-L180

The overridden group caused to publish Armeria artifacts to `Verification` org in Sonatype snapshot storage.
https://github.com/line/armeria/actions/runs/5010142242/jobs/8979748543#step:8:1273

Modifications:

- Override the group of a test task in `testTask.configure` block.

Result:

Fixed unintentionally overridden Gradle project group information.

### CODE REVIEW

**Review by trustin — APPROVED** [review]

File: kotlin/build.gradle.kts [hunk 1]
```diff
@@ -22,8 +22,15 @@ testing {
         val testNg by registering(JvmTestSuite::class) {
             useTestNG()
 
-            group = "Verification"
-            description = "Runs the TestNG unit tests"
+
+            targets {
```
Comment (trustin) [inline kotlin/build.gradle.kts:26]: `armeria/kotlin/build.gradle.kts:25:1 Needless blank line(s) (no-consecutive-blank-lines)`

```suggestion
            targets {
```

**Review by jrhee17 — APPROVED** [review]
Thanks for the quick fix @ikhoon ! 👍 👍 👍

**Review by minwoox — APPROVED** [review]
👍 👍 👍

**Comment by ikhoon** [comment]
Thanks for the quick review. 🙇‍♂️

**Comment by ikhoon** [comment]
The publishing task successfully finished.
image

**Pull request merged**